src/components/Sticky/Sticky.tsx

```diff
@@ -30,8 +30,14 @@ export default class Sticky extends React.Component<Props, State> {
   private stickyNode: HTMLElement | null = null;
 
   componentDidMount() {
-    const {stickyManager} = this.context.polaris;
-    const {boundingElement, offset, disableWhenStacked} = this.props;
+    const {
```

Comment:
The types were originally incorrect and weren't compatible with `registerStickyItem` so I added defaults and an early return